### PR TITLE
label component

### PR DESCRIPTION
**Pull request by naremanhilles**
Relates #13

### CODE REVIEW

**Comment by AbdallahAmmar96** [comment]
add a new line at the end of the file

File: Client/src/components/common/label.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+function Label(props){
+return(
+    <label htmlFor = {props.id} id={props.id}>
+    {props.labelText}
```
Comment (AbdallahAmmar96) [inline Client/src/components/common/label.jsx]: you can do destruture object 
const { id, labelText } = props
befor the return line

**Review by kannanhassouna — CHANGES_REQUESTED** [review]
the style of component

File: Client/src/components/common/label.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+function Label(props){
+return(
```
Comment (kannanhassouna) [inline Client/src/components/common/label.jsx]: Indentation

**Review by kannanhassouna — CHANGES_REQUESTED** [review]

File: Client/src/components/common/label.jsx (hunk 2 above)
Comment (naremanhilles) [reply Client/src/components/common/label.jsx]: > Indentation

edit it :+1:

File: Client/src/components/common/Label/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import "./index.css";
+function Label(props) {
+  return (
```
Comment (AbdallahAmmar96) [inline Client/src/components/common/Label/index.js:4]: add new line after 2

File: Client/src/components/common/date-picker/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+function Datepicker() {
+  return (
```
Comment (AbdallahAmmar96) [inline Client/src/components/common/date-picker/index.js:3]: add a new line after 1

Comment (AbdallahAmmar96) [inline Client/src/components/common/date-picker/index.js:3]: change Datepicker to DatePicker

File: Client/src/components/common/date-picker/index.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+    <select>
+      <option value="0">Saturday-Monday-Wednesday</option>
+      <option value="1">Sunday-Tuesday-Thursday</option>
+    </select>
+  );
+}
+export default Datepicker;
```
Comment (AbdallahAmmar96) [inline Client/src/components/common/date-picker/index.js:10]: change Datepicker to DatePicker

File: Client/src/components/common/date-picker/index.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+function Datepicker() {
+  return (
+    <select>
+      <option value="0">Saturday-Monday-Wednesday</option>
+      <option value="1">Sunday-Tuesday-Thursday</option>
+    </select>
+  );
```
Comment (AbdallahAmmar96) [inline Client/src/components/common/date-picker/index.js:8]: make this option a component so we can reuse it

File: Client/src/components/common/Label/index.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+    <label htmlFor={props.id} id={props.id} className="label">
+      {props.labelText}
+    </label>
+  );
+}
```
Comment (AbdallahAmmar96) [inline Client/src/components/common/Label/index.js:9]: you can make a destruction 
const { id, labelText } = props

**Review by AbdallahAmmar96 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**